modules/gapi/test/gapi_fluid_resize_test.cpp

```diff
@@ -829,7 +829,11 @@ TEST_P(Preproc4lpiTest, Test)
     cv::cvtColor(in_mat, rgb_mat, cv::COLOR_YUV2RGB_NV12);
     cv::resize(rgb_mat, out_mat_ocv, out_sz, 0, 0, interp);
 
+#if defined(__arm__) || defined(__aarch64__)
+    EXPECT_EQ(2, cvtest::norm(out_mat(roi), out_mat_ocv(roi), NORM_INF));
+#else
     EXPECT_EQ(0, cvtest::norm(out_mat(roi), out_mat_ocv(roi), NORM_INF));
+#endif
```

Comment:
Is it the only test which fails? Don't we instantiate other tests with this Resize?

Just asking

Reply:
> `EXPECT_EQ(2,`

`EXPECT_GE(2, ...);` or even `EXPECT_LE(..., 2);` should be better.